fact_extractor/plugins/unpacking/uefi/install.sh

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
-
-cd "$( dirname "${BASH_SOURCE[0]}" )"
-
-echo "------------------------------------"
-echo "        install uefi parser         "
-echo "------------------------------------"
-
-cd ../../../install
-
-git clone https://github.com/theopolis/uefi-firmware-parser.git
```

Comment:
So this was resolved in the meantime?

Reply:
Yes, apparently there were some major updates to `uefi-firmware-parser` also with new packages on PyPI which makes the installation much easier

Reply:
I can expand on this: Though pypi lists the package on version 1.10 and there is a correspondin release from January 5th, when installing with pip it does not recognize 1.10 and lists 1.9 as latest version. That version is not running on python3 seemingly (details, details, nevermind) so that the plugin also can not work correctly.

A possible fix would be to install `git+https://github.com/theopolis/uefi-firmware-parser@v1.10`.

Reply:
That is interesting. If I remember correctly, the test failed for me, too, but just once. After that I could not reproduce it anymore. There could also be a problem with the python version as https://pypi.org/project/uefi-firmware/ lists the package to be python 3.6/7 compatible but no newer versions. I will change it to `git+https://github.com/theopolis/uefi-firmware-parser@v1.10`. At least it seems to install without issues in the container that way.